docs/content/bundles.md

```diff
@@ -51,10 +50,10 @@ services:
         token: "bGFza2RqZmxha3NkamZsa2Fqc2Rsa2ZqYWtsc2RqZmtramRmYWxkc2tm"
 
 bundle:
-  name: authz/bundle.tar.gz
-  prefix: somedir
-  service: acmecorp
-  polling:
+  authz:
+    service: acmecorp
+    resource: somedir/authz/bundle.tar.gz
```

Comment:
Can we remove "authz" from the resource? I was a bit confused when I read the paragraph below on behaviour when "resource" isn't specified. Also, it might be good to mention that the name can contain slashes.

Reply:
Done, the updated version should hopefully be a little bit less confusing.